src/test/java/com/amazonaws/encryptionsdk/kms/KmsMasterKeyTest.java

```diff
@@ -59,38 +55,10 @@
     private static final String AWS_KMS_PROVIDER_ID = "aws-kms";
     private static final String OTHER_PROVIDER_ID = "not-aws-kms";
 
-    private static final CryptoAlgorithm ALGORITHM_SUITE = CryptoAlgorithm.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA256;
+    private static final CryptoAlgorithm ALGORITHM_SUITE = CryptoAlgorithm.ALG_AES_192_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384;
     private static final SecretKey DATA_KEY = new SecretKeySpec(generate(ALGORITHM_SUITE.getDataKeyLength()), ALGORITHM_SUITE.getDataKeyAlgo());
     private static final List<String> GRANT_TOKENS = Collections.singletonList("testGrantToken");
     private static final Map<String, String> ENCRYPTION_CONTEXT = Collections.singletonMap("myKey", "myValue");
-    private static final String CMK_ARN = "arn:aws:kms:us-east-1:999999999999:key/01234567-89ab-cdef-fedc-ba9876543210";
-    @Mock AwsKmsDataKeyEncryptionDao dataKeyEncryptionDao;
-
-    /**
-     * Test that when decryption of an encrypted data key throws a MismatchedDataKeyException, this
-     * key is skipped and another key in the list of keys is decrypted.
-     */
-    @Test
-    void testMismatchedDataKeyException() {
```

Comment:
Why was this test removed?

Reply:
Ah, it looks like this test fails. Adding back in and fixing. The issue here is that the
```
                if (!edkKeyId.equals(id_)) {
                    continue;
                }
```

Check fails, since the `edkKeyID` is `key1` or `key2` which doesn't match the `id_` (which has a CMK ARN)